source/extensions/transport_sockets/tls/context_impl.cc

```diff
@@ -751,8 +891,16 @@ SslStats ContextImpl::generateStats(Stats::Scope& store) {
 size_t ContextImpl::daysUntilFirstCertExpires() const {
   int daysUntilExpiration = Utility::getDaysUntilExpiration(ca_cert_.get(), time_source_);
   for (auto& ctx : tls_contexts_) {
-    daysUntilExpiration = std::min<int>(
-        Utility::getDaysUntilExpiration(ctx.cert_chain_.get(), time_source_), daysUntilExpiration);
+    if (ctx.cert_ == nullptr) {
+      return 0;
+    }
+    auto* start = CRYPTO_BUFFER_data(ctx.cert_.get());
+    bssl::UniquePtr<X509> x509(d2i_X509(nullptr, &start, CRYPTO_BUFFER_len(ctx.cert_.get())));
+    if (!x509) {
+      return 0;
```

Comment:
`continue`? Can this actually happen? Can we validate that this doesn't fail when setting up the ContextImpl with this cert/chain?

Reply:
Done.